EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: <to be assigned>
+title: Location Metadata Standard in ERC-721 Tokens
+author:
+Ricardo Chacon (@RicardoChacon)
```

Comment:
Please place authors on a single line, comma separated.

Reply:
See EIP-1 if you're unsure how to format.

Reply:
Formatted correctly